### ISSUE TITLE
the segmentation

### ISSUE THREAD

**Issue by woqiaow** [zhaozhengChen/ReCAM#16]
Hello, I trained the segmentation network according to the link to deeplabv2 you gave me, using The train_aug.txt (10582) pseudo mask trained the segmentation network, and then verified with val.txt (1449), the result was only 0.03. Among the 21 classes, only the background is recognized, and the results of other classes are 0. Do I need to use a pseudo mask when verifying?

**Comment by zhaozhengChen** [comment]
No, you should evaluate your segmentation model with the ground truth mask of the validation set.
When you train the segmentation network, what you need to do is replace the train_aug (10582) ground truth label with pseudo masks.

**Comment by woqiaow** [comment]
Yes, I did, but the result was very poor. I want to ask if the pre training model you used when training the segmentation network is deeplabv1_ resnet101-imagenet？

**Comment by zhaozhengChen** [comment]
Yes. Have you tested the mIoU of your pseudo mask?

**Comment by woqiaow** [comment]
yes, the pseudo mask mIoU is 69.9%.

**Comment by zhaozhengChen** [comment]
Then it shouldn't be...
Is the training loss of the segmentation model normal (not NaN)?

**Comment by woqiaow** [comment]
Yes, the problem of loss has been solved. The result of deeplabv2 mentioned in the paper is 68.5%. Is it the result of crf processing?

**Comment by zhaozhengChen** [comment]
Yes, it should be 66%-67% before crf.

**Comment by woqiaow** [comment]
Okay, thank you. I'll try running again

**Comment by zhaozhengChen** [comment]
You may also try the coco pretrained model, the training process would be more stable.

**Comment by woqiaow** [comment]
OK, I will try, thank you!

**Comment by woqiaow** [comment]
Hello, I found that the result of using train(1464) to split the network for supervision training is 63%, but the result of using train_aug(10582) to split the network for supervision training is only 3%. Please tell me whether DATASET uses voc or vocaug in the voc.yaml file during training.

**Comment by zhaozhengChen** [comment]
vocaug

**Comment by zbb1111** [comment]
I have the same problem, have you solved it?
